### PR TITLE
Remove old dead code for glade in delayed fetch

### PR DESCRIPTION
**Pull request by bradfrizzell**

### CODE REVIEW

File: ads/google/doubleclick.js [hunk 1]
```diff
@@ -55,18 +44,15 @@ export function doubleclick(global, data) {
 
   centerAd(global);
 
-  const gptFilename = selectGptExperiment(data);
-
-  writeAdScript(global, data, gptFilename);
+  writeAdScript(global, data);
```
Comment (mlb7687) [inline ads/google/doubleclick.js]: Maybe it makes sense to inline this function now. It is pretty small after this clean up.

File: ads/google/doubleclick.js [hunk 2]
```diff
@@ -55,18 +44,15 @@ export function doubleclick(global, data) {
...
 
 /**
  * @param {!Window} global
  * @param {!Object} data
- * @param {!GladeExperiment} gladeExperiment
- * @param {!string} url
  */
-function doubleClickWithGpt(global, data, gladeExperiment, url) {
+function doubleClickWithGpt(global, data) {
+  const url = 'https://www.googletagservices.com/tag/js/gpt.js';
```
Comment (mlb7687) [inline ads/google/doubleclick.js]: Nit: Moving this definition closer to the usage or inlining avoids someone having to search around for this variable.

Comment (bradfrizzell) [reply ads/google/doubleclick.js]: just removed the declaration and inlined it below

File: ads/google/doubleclick.js (hunk 1 above)
Comment (bradfrizzell) [reply ads/google/doubleclick.js]: left it not inlined for ease of not rewriting too much testing, as all the tests of that file are testing that function.

File: ads/google/test/test-doubleclick.js [hunk 3]
```diff
@@ -65,57 +49,14 @@ describes.sandboxed('writeAdScript', {}, env => {
   it('should use GPT and opt out of the GladeExperiment when' +
```
Comment (mlb7687) [inline ads/google/test/test-doubleclick.js]: References to "GladeExperiment" can be cleaned up in test cases.

File: ads/google/test/test-doubleclick.js [hunk 4]
```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import {selectGptExperiment, writeAdScript} from '../doubleclick';
+import {writeAdScript} from '../doubleclick';
 import {createServedIframe} from '../../../testing/iframe';
 
 function verifyScript(win, name) {
```
[the remark this answers is not in the document]
Comment (bradfrizzell) [reply ads/google/test/test-doubleclick.js]: done

File: ads/google/test/test-doubleclick.js (hunk 3 above)
Comment (bradfrizzell) [reply ads/google/test/test-doubleclick.js]: done

File: ads/google/doubleclick.js [hunk 5]
```diff
@@ -55,18 +44,14 @@ export function doubleclick(global, data) {
 
   centerAd(global);
```
Comment (keithwrightbos) [inline ads/google/doubleclick.js]: Remove extra line here

File: ads/google/doubleclick.js [hunk 6]
```diff
@@ -82,105 +67,99 @@ function doubleClickWithGpt(global, data, gladeExperiment, url) {
...
-        pubads.markAsGladeOptOut();
-      }
-
-      if (data['experimentId']) {
-        const experimentIdList = data['experimentId'].split(',');
-        pubads.forceExperiment = pubads.forceExperiment || function() {};
-        experimentIdList &&
+          if (data['experimentId']) {
+            const experimentIdList = data['experimentId'].split(',');
+            pubads.forceExperiment = pubads.forceExperiment || function() {};
```
Comment (keithwrightbos) [inline ads/google/doubleclick.js]: pubads.forceExperiment = pubads.forceExperiment || () => {};

File: ads/google/doubleclick.js [hunk 7]
```diff
@@ -82,105 +67,99 @@ function doubleClickWithGpt(global, data, gladeExperiment, url) {
...
+            experimentIdList &&
```
Comment (keithwrightbos) [inline ads/google/doubleclick.js]: Let's wrap this in proper if or parens (e.g. (experimentIdList || []).forEach OR if (experimentIdList) experimentIdList.forEach

File: ads/google/doubleclick.js [hunk 8]
```diff
@@ -82,105 +67,99 @@ function doubleClickWithGpt(global, data, gladeExperiment, url) {
...
-        if (event.isEmpty ||
+            if (event.isEmpty ||
             returnedSize && (rWidth > pWidth || rHeight > pHeight)) {
-          global.context.noContentAvailable();
-          creativeId = '_empty_';
-        } else {
+              global.context.noContentAvailable();
+              creativeId = '_empty_';
+            } else {
           // We only want to call renderStart with a specific size if the
```
Comment (keithwrightbos) [inline ads/google/doubleclick.js]: Ident comments as well

File: ads/google/doubleclick.js [hunk 9]
```diff
@@ -253,45 +229,19 @@ function centerAd(global) {
...
-  `https://www.googletagservices.com/tag/js/${gptFilename || 'gpt.js'}`;
-  if (gptFilename || data.useSameDomainRenderingUntilDeprecated != undefined
-    || data.multiSize) {
-    doubleClickWithGpt(global, data, GladeExperiment.GLADE_OPT_OUT, url);
+export function writeAdScript(global, data) {
+  if (data.useSameDomainRenderingUntilDeprecated != undefined
+      || data.multiSize || (
+          global.context && global.context.location &&
+            global.context.location.href &&
+            global.context.location.href.indexOf('google_glade=0') > 0 &&
```
Comment (keithwrightbos) [inline ads/google/doubleclick.js]: This really should have used location.search as well as a regexp extract IMO but I suppose you're just copying over existing...

**Pull request merged**